admin/views/tabs/social/facebook.php

```diff
@@ -34,9 +34,21 @@
 	$yform->textinput( 'og_frontpage_desc', __( 'Description', 'wordpress-seo' ) );
 
 	// Offer copying of meta description.
-	echo '<input type="hidden" id="meta_description" value="', WPSEO_Options::get( 'metadesc-home-wpseo' ), '" />';
-	echo '<p class="label desc"><a href="javascript:;" onclick="wpseoCopyHomeMeta();" class="button">', esc_html__( 'Copy home meta description', 'wordpress-seo' ), '</a></p>';
+	$homepage_meta_description = WPSEO_Options::get( 'metadesc-home-wpseo' );
+	if ( ! empty( $homepage_meta_description ) ) {
+		$copy_home_meta_desc_help = new WPSEO_Admin_Help_Panel(
+			'copy-home-meda-desc',
+			__( 'Help on copying the home meta description', 'wordpress-seo' ),
+			__( 'Click the &#8220;Copy home meta description&#8221; button to use the meta description already set in the Search Appearance homepage setting.', 'wordpress-seo' )
```

Comment:
Could you add `esc_html` to this translation?

Reply:
I haven't added escaping to the help panel content because it could contain HTML, for example: links. Open to suggestions.